docs/guide.md

```diff
@@ -300,6 +303,15 @@ If you're working on web services and want to access them locally, code-server
 can proxy to any port using either a subdomain or a subpath, allowing you to
 securely access these services using code-server's built-in authentication.
 
+By default, ports running on the same machine as code-server can be accessed at
+{current_url}/proxy/{port}. For instance, if you have code-server running on
+localhost:8080 and a Python server running on localhost:8000, you could access
+it via http://localhost:8080/proxy/8000
```

Comment:
Sorry to nit this so much, I think we might want to change this up because in this paragraph we talk about the subpath and how you can access at `/proxy/{port}`, then we talk about the subdomain version of the proxy, then we talk about the subpath again and mention `/proxy/{port}` again and I think it might not be clear why we talk about it twice.

What if we combine this with the `using a subpath` section to be something like:

```
Simply browse to `/proxy/<port>/`.  For instance, if you have code-server exposed on `domain.tld` and a Python server running on port 8080 of the same machine code-server is running on, you could access the Python server via `domain.tld/proxy/8000`.
```

And in the subdomain section maybe we do something similar and add:

```
For instance, if you have code-server exposed on `domain.tld` and a Python server running on port 8080 of the same machine code-server is running on, you could run code-server with `--proxy-domain domain.tld` and access the Python server via `http://8080.domain.tld`.
```

That way we have matching examples for both types of proxies.

Reply:
Great ideas! Let me do another version!